entrypoint.sh

```diff
@@ -60,17 +60,17 @@ if [ "$INPUT_CONTAINER_PUSH_ENABLED" = 'true' ]; then
   ./actions-collection/scripts/ecr_create.sh "$INPUT_ECR_REPOSITORY"
   echo "End: Checking ECR Repo"
   echo "Start: Publish Image to ECR"
-  ./actions-collection/scripts/publish.sh
+  pwsh ./actions-collection/scripts/publish.ps1
   echo "End: Publish Image to ECR"
 fi
 
-echo "Nuget Publish: $INPUT_NUGET_PUSH_ENABLED"
-if [ "$INPUT_NUGET_PUSH_ENABLED" = 'true' ]; then
-  echo "Start: Publish Nuget Package"
-  /scripts/nuget_push.sh
-  echo "End: Publish Nuget Package"
-fi
-
+# echo "Nuget Publish: $INPUT_NUGET_PUSH_ENABLED"
```

Comment:
Why are we no longer doing nuget push?

Reply:
I commented for easy testing. I will uncomment once testing is done

Reply:
ok Please don't forget to uncomment this